### PR TITLE
custom network api script

### PR DESCRIPTION
**Pull request by guvishwa**
create, export, and delete custom networks

create custom networks based on the CSV data

CSV file format:

name,type,ip_range,vlan_id,duplicated,split_devices_per_sensor
Network1,OT Internal,10.0.0.0/24,1,False,True
Network3,IT Internal,172.16.0.0/16,2003,False,True
Network2,IT Internal,192.168.1.0/24,1001,True,False

### CODE REVIEW

File: custom-networks.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
+import csv
```
Comment (kyechcha) [inline custom-networks.py:1]: Like in device.py, can we add some header content (check lines 1-4 in device.py

File: custom-networks.py [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        with open(csv_file, "w", encoding=csv_encoding) as file:
+            file.write(csv_data)
+        print(csv_data)
+        return
+    parser.print_help()
+
+
+def print_csv_file_format():
+    print("Valid CSV file format example")
+    print("name,type,ip_range,vlan_id,duplicated,split_devices_per_sensor")
```
Comment (kyechcha) [inline custom-networks.py:85]: Maybe we can add one additional info 

print("Note: In case name has a comma, then, provide the name in double quotes")

**Review by kyechcha — COMMENTED** [review]
Looks good. Have given some minor comments. Kindly check.

File: custom-networks.py (hunk 1 above)
Comment (guvishwa) [reply custom-networks.py:5]: Fixed

File: custom-networks.py (hunk 2 above)
Comment (guvishwa) [reply custom-networks.py:89]: Fixed

**Review by kyechcha — APPROVED** [review]
Approved

**Pull request merged**